trunk_lexer/src/lexer.rs

```diff
@@ -904,17 +911,16 @@ mod tests {
             TokenKind::OpenTag($kind)
         };
     }
-    macro_rules! var {
-        ($v:expr) => {
-            TokenKind::Variable($v.into())
-        };
-    }
     macro_rules! int {
         ($i:expr) => {
             TokenKind::Int($i)
         };
     }
 
+    fn var<B: ?Sized + AsRef<[u8]>>(v: &B) -> TokenKind {
```

Comment:
I found it was actually easier to write this as a generic function than a macro, when it needs to deal with both string and byte literals.

Reply:
This is fine. The macros were just originally a quick and dirty way of removing repetition, most of them could probably be removed so that the tests are explicit anyway.